components/ContactCard/ContactCard.module.scss

```diff
@@ -59,18 +77,28 @@
     width: 1rem;
     height: 1rem;
     flex-shrink: 0;
-    margin-right: $spacing-1;
 }
 
 .container {
-    @include border-radius-lg;
-
-    padding: $spacing-4;
     background: $color-base-50;
```

Comment:
I guess the background should also be omitted for signature layout?

Reply:
Possibly but I haven't noticed any difference by removing this rule 🤔